Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -1419,4 +1427,78 @@ - (void)serverCallback:(BNCServerResponse *)response {
     }
 }
 
+#pragma mark - Debugging functions
+
+- (void)bnc_addDebugGestureRecognizer {
+    [self bnc_addGesterRecognizer:@selector(bnc_connectToDebug:)];
+}
+
+- (void)bnc_addCancelDebugGestureRecognizer {
+    [self bnc_addGesterRecognizer:@selector(bnc_endDebug:)];
+}
+
+- (void)bnc_addGesterRecognizer:(SEL)action {
+    BNCLongPress = [[UILongPressGestureRecognizer alloc] initWithTarget:self action:action];
```

Comment:
I'm not 100% sure how this works, but if you're adding this to the window, I believe that other gesture recognizers added by the app developers will be ignored. I'm not sure if the "cancelsTouchesInView" will pass through and call both yours and theirs or not.